test_actors/actors/constructor_test_actor/src/actor.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Polyphene.
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::{multihash::Code, Cid};
+use fvm_ipld_blockstore::Block;
+use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+use fvm_ipld_encoding::DAG_CBOR;
+use fvm_sdk as sdk;
+use fvm_sdk::NO_DATA_BLOCK_ID;
+use fvm_shared::error::ExitCode;
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+struct TestConstructorActorState {
+    value: u32,
+}
+
+impl TestConstructorActorState {
+    pub fn load(cid: &Cid) -> Self {
+        let data = sdk::ipld::get(cid).unwrap();
+        fvm_ipld_encoding::from_slice::<Self>(&data).unwrap()
+    }
+
+    pub fn save(&self) -> Cid {
+        let serialized = fvm_ipld_encoding::to_vec(self).unwrap();
+        let block = Block {
+            codec: DAG_CBOR,
+            data: serialized,
+        };
+        sdk::ipld::put(
+            Code::Blake2b256.into(),
+            32,
+            block.codec,
+            block.data.as_ref(),
+        )
+        .unwrap()
+    }
+}
+
+// TODO use helix frc42_dispatch when their dependencies are up to date.
+#[no_mangle]
+fn invoke(_input: u32) -> u32 {
+    let method_num = sdk::message::method_number();
+    match method_num {
```

Comment:
We should leverage `frc42_dispatch` crate here I think. See https://github.com/polyphene/kythera/blob/develop/test_actors/actors/builtins_test_actor/src/actor.rs#L26-L41

Reply:
removed this one as the test was also removed making it redundant